src/sentry/static/sentry/app/views/ruleEditor/ruleNode.jsx

```diff
@@ -20,31 +23,17 @@ class RuleNode extends React.Component {
       $el.attr('id', '');
       $el.val(this.props.data[el.name]);
```

Comment:
Unfortunate that it's still setting values like this :( I figure we'll address this in the next pass right?

Reply:
Rewriting the entire component isn't in scope of this PR. I can come back to it later, but this is mostly about changing the API response.